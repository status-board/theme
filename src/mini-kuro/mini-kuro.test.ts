import miniKuro from './index';

test('Mini Kuro should match snapshot', (): void => {
    expect(miniKuro).toMatchSnapshot();
});
