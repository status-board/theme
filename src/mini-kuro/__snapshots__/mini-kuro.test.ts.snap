// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mini Kuro should match snapshot 1`] = `
Object {
  "charts": Object {
    "axis": Object {
      "legendColor": "#fff",
      "textColor": "#fff",
      "tickColor": "#fff",
    },
    "colors": Array [
      "#66c2a5",
      "#fc8d62",
      "#8da0cb",
      "#e78ac3",
      "#a6d854",
    ],
    "grid": Object {
      "stroke": "#222",
    },
    "tooltip": Object {
      "background": "#000",
      "border": "1px solid #fff",
      "borderRadius": 0,
      "boxShadow": "none",
      "color": "#fff",
      "fontSize": "1.4vmin",
    },
  },
  "colors": Object {
    "failure": "#ff9176",
    "icon": "#fff",
    "success": "#4eb6a3",
    "text": "#fff",
    "textHighlight": "#fff",
    "unknown": "#c0ab7f",
    "warning": "#d1be65",
  },
  "dashboard": Object {
    "header": Object {
      "title": Object {},
    },
  },
  "label": Object {
    "addon": Object {
      "extend": "
            font-weight: 700;
            &:first-child {
                padding-right: 1.4vmin;
                border-right: 1px dotted #fff;
            }
            &:last-child {
                padding-left: 1.4vmin;
                border-left: 1px dotted #fff;
            }
            & + .Label {
                padding-left: 1.4vmin;
            }
            .Label + & {
                margin-left: 1.4vmin;
            }
            ",
      "padding": "1vmin 0.6vmin",
    },
    "extend": "
        line-height: 1em;
        border-top: 1px dotted #fff;
        border-bottom: 1px dotted #fff;
        align-items: baseline;
        ",
    "main": Object {
      "padding": "1vmin 0.2vmin",
    },
  },
  "list": Object {
    "item": Object {
      "extend": "
            border-bottom: 1px solid #fff;
            &:last-child {
                border-bottom: 0;
            }
            ",
      "meta": Object {
        "extend": "
                font-weight: 300;
                ",
        "fontSize": "1.4vmin",
      },
      "padding": "1vmin 0",
    },
  },
  "name": "mini kuro",
  "notifications": Object {
    "item": Object {
      "background": "#fff",
      "color": "#000",
      "extend": "
            font-weight: 400;
            ",
      "padding": "1.2vmin 2vmin",
    },
  },
  "root": Object {
    "background": "#000",
    "extend": "
        @import url('https://fonts.googleapis.com/css?family=Roboto+Mono:300,400,500,700');
        font-weight: 300;
        
        & a {
            font-weight: 400;
        }
        & img {
            filter: grayscale(100%);
        }
        ",
  },
  "typography": Object {
    "default": Object {
      "default": Object {
        "fontFamily": "'Roboto Mono', monospace",
        "fontSize": "1.8vmin",
        "fontWeight": 400,
        "lineHeight": "1.5em",
      },
      "small": Object {
        "fontSize": "1.6vmin",
      },
      "strong": Object {
        "fontWeight": 600,
      },
    },
    "display": Object {
      "default": Object {
        "fontFamily": "'Roboto Mono', monospace",
        "fontSize": "1.8vmin",
        "fontWeight": 700,
        "lineHeight": "1.5em",
      },
    },
  },
  "widget": Object {
    "background": "#000",
    "header": Object {
      "count": Object {
        "extend": "
                font-weight: 700;
                vertical-align: baseline;
                ",
      },
      "extend": "
            border-top: 2px solid #fff;
            align-items: flex-start;    
            ",
      "icon": Object {
        "extend": "
                display: none;
                ",
      },
      "padding": "0.8vmin 0 0 0",
      "subject": Object {},
    },
    "wrapper": Object {
      "padding": "1.6vmin 3vmin",
    },
  },
}
`;
